airbyte-cdk/bulk/toolkits/load-low-code/src/test/kotlin/io/airbyte/cdk/load/lowcode/DeclarativeDestinationFactoryTest.kt

```diff
@@ -301,9 +317,309 @@ class DeclarativeDestinationFactoryTest {
         assertEquals(expectedOperations, actualOperations)
     }
 
+    @Test
+    internal fun `test dynamic operations with static destination objects`() {
+        val config =
+            Jsons.readTree("""{"api_id": "$VALID_API_ID", "api_token": "$VALID_API_TOKEN"}""")
+
+        val mockedResponse =
+            mapOf<String, Any>(
+                "properties" to
+                    listOf(
+                        mapOf("name" to "name", "type" to "string"),
+                        mapOf("name" to "breathingForm", "type" to "art"),
+                        mapOf("name" to "age", "type" to "numberlike")
+                    )
+            )
+
+        val objectsResponse: Response = mockk<Response>()
+        every { objectsResponse.statusCode } returns 200
+        every { objectsResponse.body } returns
+            createObjectsResponse(mockedResponse).byteInputStream(Charsets.UTF_8)
+        every { objectsResponse.close() } returns Unit
+
+        mockkConstructor(HttpRequester::class)
+        every { anyConstructed<HttpRequester>().send(any()) } returns objectsResponse
+
+        mockManifest(
+            ManifestBuilder()
+                .withChecker(
```

Comment:
nit: Can we remove `withChecker` for this test? It feels like it would be more expressive if the manifest builder would only need to mention what is required for this test

Reply:
yep not a problem. i'll get rid of it. i agree better to make the test check a specific aspect